public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import {
+  EuiButton,
+  EuiEmptyPrompt,
+  EuiLink,
+  EuiIcon,
+  EuiPage,
+  EuiTitle,
+  EuiPageSideBar,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiSideNav,
+} from '@elastic/eui';
+import React, { Component, Fragment } from 'react';
+import { APP_PATH, PLUGIN_NAME } from '../../../../utils/constants';
+
+type EmptyDashboardState = {
+  selectedItemName: string;
+};
+
+export class EmptyDashboard extends Component<{}, EmptyDashboardState> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      selectedItemName: 'Lion stuff',
+    };
+  }
+
+  selectItem = (name: string) => {
+    this.setState({
+      selectedItemName: name,
+    });
+  };
+
+  createItem = (name: string, id: number, data = {}) => {
+    // NOTE: Duplicate `name` values will cause `id` collisions.
+    return {
+      ...data,
+      id: name,
+      name,
+      isSelected: this.state.selectedItemName === name,
+      onClick: () => this.selectItem(name),
+    };
+  };
+
+  render() {
+    const sideNav = [
+      {
+        name: 'Anomaly detection',
+        id: 0,
+        items: [
+          this.createItem('Dashboard', 1, { href: `#${APP_PATH.DASHBOARD}` }),
+          this.createItem('Detectors', 2, {
+            href: `#${APP_PATH.LIST_DETECTORS}`,
+          }),
+        ],
+      },
+    ];
+
+    return (
+      <EuiPage style={{ flex: 1 }}>
+        <EuiPageSideBar
+          style={{
+            flex: 1,
+            backgroundColor: '#F5F7FA',
+          }}
+        >
+          <EuiSideNav style={{ width: 150 }} items={sideNav} />
+        </EuiPageSideBar>
+        <EuiPageBody>
+          <EuiPageHeader>
+            <EuiTitle size="l">
+              <h1>Dashboard</h1>
+            </EuiTitle>
+          </EuiPageHeader>
```

Comment:
I think you can move `sidebar` and `Dashboard` header to `Dashboard.tsx`, because those 2 components will be reused even in Dashboard with detectors.

Reply:
good idea.  moved to a util class since I don't want EmptyDashboard to reference components defined in Dashboard.tsx as there would be circular dependency.  Dashboard.tsx references EmptyDashboard now.

Reply:
hmmm, my previous thought is: `Dashboard.tsx` has `sidebar` + `header` + `EmptyDashboardMessage`, and `EmptyDashboardMessage` just returns <EuiPageBody> with all the messages inside.  If there exists detectors: `Dashboard.tsx` will be `sidebar` + `header` + `ActiveDetectorsDashboard`.

Given the latest commit, since we just need to have `sidebar` + `header` in the future for active dashboard, it is fine with me. I will approve it.